scripts/Logistics-Pipes.zs

```diff
@@ -0,0 +1,873 @@
+
+// --- Remote Orderer Logistics Pipe
+Assembler.addRecipe(<LogisticsPipes:item.PipeItemsRemoteOrdererLogistics>, [<LogisticsPipes:item.remoteOrdererItem:*> * 0, <LogisticsPipes:item.PipeItemsRequestLogistics>, <gregtech:gt.integrated_circuit:18> * 0], null, 600, 30);
+
+
+// Until I get a Better Suggestion for theese it is just chest + plates and previous tier
+// --- Logistics Chassi MK1
+Assembler.addRecipe(<LogisticsPipes:item.PipeLogisticsChassiMk1>, [<LogisticsPipes:item.PipeItemsBasicLogistics>, <ore:plateRedstoneAlloy> * 4, <ore:chestCopper>, <gregtech:gt.integrated_circuit:18> * 0], null, 600, 30);
+// --- Logistics Chassi MK2
+Assembler.addRecipe(<LogisticsPipes:item.PipeLogisticsChassiMk2>, [<LogisticsPipes:item.PipeLogisticsChassiMk1>, <ore:plateDarkSteel> * 4, <ore:chestIron>, <gregtech:gt.integrated_circuit:18> * 0], null, 600, 30);
+// --- Logistics Chassi MK3
+Assembler.addRecipe(<LogisticsPipes:item.PipeLogisticsChassiMk3>, [<LogisticsPipes:item.PipeLogisticsChassiMk2>, <ore:plateInfusedGold> * 4, <ore:chestSteel>, <gregtech:gt.integrated_circuit:18> * 0], null, 600, 30);
+// --- Logistics Chassi MK4
+Assembler.addRecipe(<LogisticsPipes:item.PipeLogisticsChassiMk4>, [<LogisticsPipes:item.PipeLogisticsChassiMk3>, <ore:plateRedSteel> * 4, <ore:chestGold>, <gregtech:gt.integrated_circuit:18> * 0], null, 600, 30);
+// --- Logistics Chassi MK5
+Assembler.addRecipe(<LogisticsPipes:item.PipeLogisticsChassiMk5>, [<LogisticsPipes:item.PipeLogisticsChassiMk4>, <ore:plateTitanium> * 2, <ore:plateNetherStar> * 2, <ore:chestDiamond>, <gregtech:gt.integrated_circuit:18> * 0], null, 600, 30);
+
+
+// LV
+// ||||| Upgrades |||||
+// --- Blank Upgrade
+Assembler.addRecipe(<LogisticsPipes:item.logisticsParts:4> * 4, [<ProjRed|Core:projectred.core.part> * 4, <ore:circuitPrimitive>, <ore:foilTin> * 4, <ore:wireFineAnyCopper> * 4, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.lead> * 144, 600, 30);
+Assembler.addRecipe(<LogisticsPipes:item.logisticsParts:4> * 4, [<ProjRed|Core:projectred.core.part> * 4, <ore:circuitPrimitive>, <ore:foilTin> * 4, <ore:wireFineAnyCopper> * 4, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.tin> * 72, 600, 30);
+Assembler.addRecipe(<LogisticsPipes:item.logisticsParts:4> * 4, [<ProjRed|Core:projectred.core.part> * 4, <ore:circuitPrimitive>, <ore:foilTin> * 4, <ore:wireFineAnyCopper> * 4, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.solderingalloy> * 36, 600, 30);
+
+// --- Fuzzy Upgrade
+CircuitAssembler.addRecipe(<LogisticsPipes:item.itemUpgrade:25>, [<LogisticsPipes:item.logisticsParts:4>, <ore:blockWool>, <ore:foilSteel>, <ProjRed|Core:projectred.core.part:26>], null, 600, 30); //
+
+// --- Item  Speed Upgrade
+CircuitAssembler.addRecipe(<LogisticsPipes:item.itemUpgrade:20>, [<LogisticsPipes:item.logisticsParts:4>, <gregtech:gt.metaitem.01:32640>, <ore:foilBeryllium>, <ProjRed|Core:projectred.core.part:32>], null, 600, 30); //

```

Comment:
Beryllium is a really weird choice here. Might use some lubricant instead.

Reply:
Will change this when i work on fluid pipes